INSTALL.txt

```diff
@@ -1,64 +1,255 @@
-Quick compilation instructions for GNU/Linux, Mac OS X, or Windows if you have
-installed MinGW or Cygwin: make sure "automake" is installed as well as the usual C
-compiler chain. Then in this directory, run the following commands:
+# Installing Pure Data
 
-    $ ./autogen.sh
-    $ ./configure
-    $ make
+This README file contains info for installing Pure Data from source.
 
-If you need jack support, add the corresponding flag to "configure":
+Pd is built on the commandline using traditional Unix-style tools. The source
+distribution comes with two build systems:
 
-    $ ./configure --enable-jack
+* autotools: easy to use, many compilation options, recommended for most users
+* makefile: smaller & simpler, used for Pd binary downloads
 
-See the ./configure help print for other options such as a multi architecture
-(universal) build, etc:
+## Requirements
 
-    $ ./configure --help
+Core build requirements:
 
-Pd can now be run out of the bin directory via:
+* Unix command shell: bash, dash, etc
+* C compiler chain: gcc/clang & make
+* Tcl/Tk: Tk wish shell
```

Comment:
technically, tcl/tk is not a build requirement (it is a requirement to *run* Pd with a GUI). this is probably slightly blurred if by "building" you mean "creating a (self-contained) package that you can put somewhere and run Pd" (e.g. a Pd.app)